pkg/reconciler/events/pubsub/pubsub.go

```diff
@@ -75,37 +63,38 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, pubsub *v1alpha1.CloudPu
 	ctx = logging.WithLogger(ctx, r.Logger.With(zap.Any("pubsub", pubsub)))
 
 	pubsub.Status.InitializeConditions()
-	pubsub.Status.ObservedGeneration = pubsub.Generation
```

Comment:
this line was correct. The previous behavior was wrong. You change corrected it. So add this one back

Reply:
got it, I misunderstood. Fixing here and in the other PR.